### ISSUE TITLE
Why is TimeSpan not a simple Type?

### ISSUE THREAD

**Issue by Piddu1985** [GregFinzer/Compare-Net-Objects#124]
If I run this code to catch a difference between two TimeSpan fields:
```
static class Program
{
    static void Main(string[] args)
    {
        var paot1 = new PAOT { OraInizio = new TimeSpan(6, 0, 0) };
        var paot2 = new PAOT { OraInizio = new TimeSpan(7, 0, 0) };

        var comparerConfig = new ComparisonConfig();
        comparerConfig.CompareChildren = false;
        var comparer = new CompareLogic(comparerConfig);
        var result = comparer.Compare(paot1, paot2);
    }

    class PAOT
    {
        public TimeSpan OraInizio { get; set; }
    }
}
```
I don't get any difference, `result.AreEqual = true`, which means that the type `System.TimeSpan` is treated as a "child" of my class, and not as a simple, "scalar" type. I indeed find it very clearly in the source code at `TypeHelper.CanHaveChildren`, so it's by design.

But I don't understand the reasoning behind this, especially since I see that, inside `TypeHelper.IsSimpleType`, there is a special exception for `System.DateTime` and I fail to see any big difference in this regard between the types. Can anybody give me an answer on that?

Thanks

**TL;DR** why is a `System.TimeSpan` property ignored when `ComparisonConfig.CompareChildren` is false?

**Comment by GregFinzer** [comment]
@Piddu1985 there is no reasoning for that.  :)  You have found a bug.  Thanks for reporting it.

**Comment by GregFinzer** [comment]
Fixed
